.mergify.yml

```diff
@@ -47,33 +47,36 @@ pull_request_rules:
       - check-failure=cmake-checks
       - check-failure=frozen-tools-check 
       - "label!=mergify skip"
+      - "label!='needs: work'"
     actions:
       label:
         add: ['needs: work']
         remove: ['needs: review', 'needs: CI']
 
-  # From needs: review to needs: work - CI failure
+  # From needs: CI to needs: work - CI failure in jenkins pipeline
   - name: "label needs: work when Jenkins CI failed - pr head"
     conditions:
-      # Jenkins CI failing
+      # Jenkins CI failing, only pr head
       - check-failure~=continuous-integration/jenkins/pr-head
       - "label!=mergify skip"
+      - "label=needs: CI"
       - -closed
     actions:
       label:
         add: ['needs: work']
-        remove: ['needs: review','needs: CI']
+        remove: ['needs: CI']
```

Comment:
Why has the remove bit changed? If there is a needs:review doesn't that also need removing ?